charts/internal/machine-controller-manager/seed/templates/vpa.yaml

```diff
@@ -11,4 +11,16 @@ spec:
     name: machine-controller-manager
   updatePolicy:
     updateMode: {{ .Values.vpa.updatePolicy.updateMode | quote }}
+  resourcePolicy:
+    containerPolicies:
+    - containerName: machine-controller-manager-provider-equinix-metal
+      maxAllowed:
+        cpu: 2
+        memory: 5G
+      controlledValues: RequestsOnly
```

Comment:
I don't think we should be switching on `controlledValues: RequestsOnly` here. We didn't adjust the Limits to the values we've actually seen in production, so telling VPA to not scale Limits is pretty dangerous here.

Reply:
Fixed